config/output.go

```diff
@@ -36,7 +36,7 @@ func (t *Config) RunOutputs() (err error) {
 func (t *Config) runOutputs(evchan chan logevent.LogEvent, logger *logrus.Logger) (err error) {
 	outputs, err := t.getOutputs()
 	if err != nil {
-		return errutil.New("get outputs failed", err)
+		return err
```

Comment:
I would agree but when running the test - without rabbitmq, elasticsearch nor redis - I saw that `errutil.New("get outputs failed", err)` would only carry `"get outputs failed"`.

See gist for the version with only `return err`: 
https://gist.github.com/mirdhyn/2bea74ddfeaf689aa6c1a1ce4ef626a8
and the one with errutil:
https://gist.github.com/mirdhyn/c519636776e4224334f1c8d7af87ed69